scripts/run-gradually-deprecated.sh

```diff
@@ -105,6 +105,12 @@ function runGraduallyDeprecatedFunctions {
         fi
     fi
 
+    # require Azure SDK clients are created with the resource manager endpoint specified 
+    grep -H -n "Client(o.SubscriptionId)" "$f" && {
```

Comment:
running this locally to investigate the error, this is failing because it's picking up this script 😬 - going to push a commit to move this up inside the `if` statement above